CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog (Pypi package)
+
+## [1.3.44] 2022-01-19
+
+### Added
+
```

Comment:
Both of these additions were on 1.3.43
I think you can add a [1.3.43] 2022-01-18 section perhaps ? Thanks !

Reply:
updated ! 👍🏾